packages/core/data-transfer/src/strapi/remote/handlers.ts

```diff
@@ -89,26 +82,49 @@ export const createTransferHandler = (options: IHandlerOptions) => {
             ws.send(payload, (error) => (error ? reject(error) : resolve()));
           });
         };
+        /**
+         * Format message to follow the remote transfer protocol
+         */
+        const sendResponse = <T = unknown>(uuid: string, data: T) => {
+          if (data instanceof Error) {
+            return sendErrorResponse(uuid, data);
+          }
+
+          return new Promise<void>((resolve, reject) => {
+            if (!uuid) {
+              reject(new Error('Missing uuid for this message'));
+              return;
+            }
+
+            const payload = JSON.stringify({
+              uuid,
+              data: data ?? null,
+            });
+
+            ws.send(payload, (error) => (error ? reject(error) : resolve()));
+          });
+        };
 
         /**
          * Wrap a function call to catch errors and answer the request with the correct format
          */
-        const answer = async <T = unknown>(fn: () => T) => {
+        const answer = async <T = unknown>(fn: () => T, uuid: string) => {
```

Comment:
Need to fix params order `answer(fn, uuid)` to correspond with `sendResponse(uuid, data)`